docs/conf.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+#
+# complexity documentation build configuration file, created by
+# sphinx-quickstart on Tue Jul  9 22:26:36 2013.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import tempfile
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# sys.path.insert(0, os.path.abspath('.'))
+
+cwd = os.getcwd()
+parent = os.path.dirname(cwd)
+sys.path.insert(0, parent)
+
+import kolibri
+
+# When we start loading stuff from kolibri, we're gonna need this
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "kolibri.deployment.default.settings.base")
+os.environ["KOLIBRI_HOME"] = tempfile.mkdtemp()
```

Comment:
You're right, I do that in the tests, I just forgot to copy-past that part to sphinx... will have a look at when it can be reasonably removed again....